src/renderer/api/kube-watch-api.ts

```diff
@@ -158,6 +172,10 @@ export class KubeWatchApi {
 
   addListener(store: KubeObjectStore, callback: (evt: IKubeWatchEvent) => void) {
     const listener = (evt: IKubeWatchEvent<KubeJsonApiData>) => {
+      if (evt.type === "ERROR") {
```

Comment:
Does this fix a bug? Should we backport it to 4.0.x?

Reply:
Maybe, but with watch-api related work for #1898 this will be removed soon.